pkg/packagekit/updates.jsx

```diff
@@ -109,23 +111,26 @@ function HeaderBar(props) {
     }
 
     var lastChecked;
-    if (props.timeSinceRefresh) {
-        lastChecked = (
-            <span style={{paddingRight: "3ex"}}>
-                {cockpit.format(_("Last checked: $0 ago"), moment.duration(props.timeSinceRefresh * 1000).humanize())}
-            </span>
-        );
+    var actionButton;
+    if (props.state == "uptodate" || props.state == "available") {
+        actionButton = <button className="btn btn-default" onClick={() => props.onRefresh()} >{_("Check for updates")}</button>;
+        if (props.timeSinceRefresh) {
+            lastChecked = (
+                <span style={{paddingRight: "3ex"}}>
```

Comment:
I think some extra whitespace here would help, since this is not just a plain function call. What do you think about
```javascript
{ {paddingRight: "3ex"} }
```